tests/bluetooth/bsim_bt/bsim_test_advx/src/main.c

```diff
@@ -247,6 +247,83 @@ static void test_advx_main(void)
 	}
 	printk("success.\n");
 
+	printk("enabling...");
+	err = ll_adv_enable(handle, 1);
+	if (err) {
+		goto exit;
+	}
+	printk("success.\n");
+
+	k_sleep(K_MSEC(400));
+
+	printk("Update advertising data 1...");
+	err = ll_adv_aux_ad_data_set(handle, AD_OP, AD_FRAG_PREF,
+				     sizeof(adv_data1), (void *)adv_data1);
+	if (err) {
+		goto exit;
+	}
+	printk("success.\n");
+
+	k_sleep(K_MSEC(400));
+
+	printk("Update advertising data 2...");
+	err = ll_adv_aux_ad_data_set(handle, AD_OP, AD_FRAG_PREF,
+				     sizeof(adv_data2), (void *)adv_data2);
+	if (err) {
+		goto exit;
+	}
+	printk("success.\n");
+
+	k_sleep(K_MSEC(400));
+
+	printk("Disabling...");
+	err = ll_adv_enable(handle, 0);
+	if (err) {
+		goto exit;
+	}
+	printk("success.\n");
+
+	printk("Removing adv aux set ...");
+	err = ll_adv_aux_set_remove(handle);
+	if (err) {
+		goto exit;
+	}
+	printk("success.\n");
+
+	k_sleep(K_MSEC(1000));
+
+	printk("enabling...");
+	err = ll_adv_enable(handle, 1);
```

Comment:
The expected result should be `BT_HCI_ERR_UNKNOWN_ADV_IDENTIFIER`

Reply:
Changed the tests.